tests/integration/sync/it_sync_location_test.py

```diff
@@ -19,70 +15,12 @@
 
 
 @pytest.mark.skip("sync for location not implemented yet")
-def test_sync_location(db_session):
-    port = free_port()
-    start_server(port,
-        locations = [
-            {
-                "id":100,
-                "name":"Coco Bongo",
-                "code":"C",
-                "company_id":50,
-                "country":"United States",
-                "region":"East Coast",
-                "city":"Edge City",
-                "address":"Blvd. Kukulcan Km 9.5 #30, Plaza Forum",
-                "longitude":21.1326063,
-                "latitude":-86.7473191,
-                "type":"L",
-                "status":"open",
-                "comment":"Nightclub from a famous movie"
-            }
-        ]
-    )
-    company = Company(
-        id=50,
-        name="Company of Heroes",
-        code="Cpny",
-        address="Somewhere in the bermuda triangle"
-    )
-    db_session.add(company)
-    db_session.commit()
-    location = Location(
-        id=100,
-        name="Coconut Bongolive",
-        code="C",
-        company_id=50,
-        country="United States of America",
-        region="West Coast",
-        city="Another city",
-        address="Some address in Another City",
-        longitude=42.2642026,
-        latitude=-172.148146,
-        type="L",
-        status="closed",
-        comment="A location with "
-    )
-    db_session.add(location)
-    db_session.commit()
-    synced_location = SyncedLocation(location).sync(
```

Comment:
@vladarefiev Shouldn't you delete SyncedLocation class, if we are going with the task alternative?

Reply:
@amihaiemil ok, i can leave it